README.md

```diff
@@ -559,6 +559,9 @@ does not produce any values, so the following search options will return an empt
 | `oclc`                                 |   term    | `oclc="1234*"`                                                       | Matches instances that have an OCLC identifier with the given value                                                                               |
 | `lccn`                                 |   term    | `lccn = "LCCN"`                                                      | Matches instances with the given lccn                                                                                                             |
 | `normalizedClassificationNumber`       |   term    | `normalizedClassificationNumber == "LCCN"`                           | Matches instances with the given classification number (normalizes case, whitespaces, special characters, supports leading and trailing wildcard) |
+| `dates.date1`                          |   term    | `dates.date1="199*"`                                                 | Matches instances with the given Date1 (supports leading, trailing  and internal wildcards)                                                       |
+| `dates.date2`                          |   term    | `dates.date2="199*"`                                                 | Matches instances with the given Date2 (supports leading, trailing  and internal wildcards)                                                       |
+| `sort_date1`                           |   term    | `sort_date1=199u`                                                    | Matches instances with the given Date1 (normalizes  alpha 'u' characters, supports leading, trailing  and internal wildcards)                     |
```

Comment:
Why not `normalizedDate1` ?

Reply:
```suggestion
| `sort_date1`                           |   term    | `sort_date1=1990`                                                    | Matches instances with the given Date1 (normalizes  alpha 'u' characters, supports leading, trailing  and internal wildcards)                     |
```
As you don't do input term normalization.

Reply:
The "sort_" prefix in the field name is required for the sortBy operation. By default, the "sort_" prefix is ​​added during the sortBy operation in CqlSortProvider, and if it cannot find a field with this prefix, we get this error: "No mapping found for [sort_normalizedDate1] in order to sort on".

- field name: "normalizedDate1"
- error:
![image](https://github.com/user-attachments/assets/f639a976-e107-44e0-b084-3214fd0f873a)

Reply:
Could we do smth similar to authority mappings?
![image](https://github.com/user-attachments/assets/960de876-f886-4a96-8a2a-eb14862be3e4)